openshift/ci/tests/e2e-ansible-scaffolding.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+source hack/lib/common.sh
+
+set -eux
+
+component="osdk-ansible-e2e"
+eval IMAGE=$IMAGE_FORMAT
+ROOTDIR="$(pwd)"
+GOTMP="$(mktemp -d -p $GOPATH/src)"
+trap_add 'rm -rf $GOTMP' EXIT
+
+mkdir -p $ROOTDIR/bin
+export PATH=$ROOTDIR/bin:$PATH
+
+# Install kubectl client
+if ! [ -x "$(command -v kubectl)" ]; then
+    curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.21.2/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl bin/
+fi
+
+# Install oc client
+if ! [ -x "$(command -v oc)" ]; then
+    curl -Lo oc.tar.gz https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
```

Comment:
Should this refer to latest openshift release?

Reply:
Sure, updated to "4.16.0"